installation/scripts/install-compass.sh

```diff
@@ -49,6 +51,31 @@ do
 done
 set -- "${POSITIONAL[@]}" # restore positional parameters
 
+echo "Install DB"
+helm upgrade --install --wait --debug --timeout "${TIMEOUT}" -f ./mergedOverrides.yaml --create-namespace --namespace compass-system postgresql "${DB_CHARTS}"
+
+echo "Look for DB migrations in progress ..."
```

Comment:
The idea is not to execute migrations in the local db chart. Only to apply the dump.

Currently, in the migrator job we have logic for:
1. Download the dump form GCP
2. Apply the dump on the local DB
3. Apply migrations from schema-migrator folder

We need to encapsulate steps 1 and 2 in a dedicated script (as a new db-dump job) in the new local db chart and the 3rd step should be left in the schema-migrator job (all dump related stuff is removed from it).

Reply:
For 1 - Not really. The Dump is downloaded in install script, not in migrator job.
For 3 - Applying the migrations on DB dump restore is Ok, and will not have any side effect, as the migrator job that is part of compass chart will check and will apply the new one (that are part of the new PR) if needed - in case on later commit the migrations were added, not on the first commit.